docs/configuration/http_conn_man/route_config/route.rst

```diff
@@ -36,12 +37,27 @@ next (e.g., redirect, forward, rewrite, etc.).
 
 prefix
   *(sometimes required, string)* If specified, the route is a prefix rule meaning that the prefix
-  must match the beginning of the :path header. Either *prefix* or *path* must be specified.
+  must match the beginning of the :path header. Either *prefix*, *path*, or *regex* must be specified.
```

Comment:
One of prefix, path or regex

Reply:
That does sound better, I'll change it :)